docker-compose.yaml

```diff
@@ -45,6 +45,8 @@ services:
       DB_NAME: ${DB_NAME}
       DB_SCHEMA: /app_sqlschema/schema.sql
       SEED_MODE: ${SEED_MODE}
+      AUTH_TYPE: token
```

Comment:
For me its fine if we keep AUTH_TYPE for the first iteration. But once we got OIDC-based authentication implemented, it needs to go away.

Reply:
Yes, we will try to create combo authenticator or separate one for user's endpoint.